CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.5.1] - 2022-11-08
+
+### Fixed
+- A regression in behavior preventing the use of default_svid_ttl in Server configuration (#3583)
```

Comment:
It would be good to state when this bug was introduced.

```suggestion
- The `default_svid_ttl` configurable is now correctly observed after fixing a regression introduced in 1.5.0 (#3583)
```

Reply:
Should we use the word `deprecated` when discussing that configurable?

Reply:
updated to `The deprecated `default_svid_ttl` configurable is now correctly observed after fixing a regression introduced in 1.5.0 (#3583)`